contrib/kubernetes/skydive.yaml

```diff
@@ -175,6 +175,10 @@ spec:
           mountPath: /var/data/kubelet
         - name: lib-kubelet
           mountPath: /var/lib/kubelet
+        - name: data-openshiftvolumes
```

Comment:
This should probably be done only in contrib/openshift : contrib/kubernetes is there for `pure` Kubernetes
Not sure what to do for Helm though. Does it work if the folder does not exist on the host ?

Reply:
Yes it works, will simply show as an empty dir.

Reply:
Ok, thanks for the clarification. So we can probably put it in the Helm chart. Still, I think this should not be done in contrib/kubernetes

Reply:
Done in b741b8a85b2d2e8b733203acef093931bee30432 (force-pushed).